unittests/kernelcache.py

```diff
@@ -0,0 +1,191 @@
+import os
+import platform
+import pytest
+import requests
+import sys
+import tempfile
+import urllib3
+import warnings
+
+try:
+    cwd = os.environ['CSPYCE_TEST_KERNELS']
+except KeyError:
+    cwd = "/tmp" if platform.system() == "Darwin" else tempfile.gettempdir()
+    
+server = 'https://pds-rings.seti.org/testrunner_support/cspyce-unit-test-kernels/'
```

Comment:
This isn't really a "server". How about something like `kernel_url_root`? But since it's really a global constant, it should be `KERNEL_URL_ROOT`.